examples/e-commerce/src/app.css

```diff
@@ -90,7 +90,7 @@ h2 {
   padding: 30px 0;
 }
 
-.container-options div:not(:first-of-type) {
+.container-options .container-option:not(:first-of-type) {
```

Comment:
This doesn't work for Angular-IS.

![image](https://user-images.githubusercontent.com/499898/60269755-5e682500-98ef-11e9-8fbc-658394bdc766.png)

Do you have any idea?
`:first-child` works, though.
I don't get the difference here.

Reply:
> Using :first-child is very similar to :first-of-type but with one critical difference: it is less specific. :first-child will only try to match the immediate first child of a parent element, while first-of-type will match the first occurrence of a specified element, even if it doesn't come absolutely first in the HTML.

Source: https://css-tricks.com/almanac/selectors/f/first-child/

Angular probably creates some sub-elements making `:first-of-type` ineffective.

Reply:
Yes according to the descriptions, it's supposed to work.
I'll change it to `:first-child`.

Reply:
Angular creates web components and I don't think we can use pseudo elements on web components. Hence, only `:first-child` works on the first child of the web component.